### ISSUE TITLE
Inconsistent Results on Cornell & Arxiv | Request for Configurations from Table 4 & 3

### ISSUE THREAD

**Issue by YanJiangJerry** [Cqkkkkkk/DAGPrompT#2]
Thank you for your excellent work! I am very interested in your research and have been trying to reproduce the node classification results. However, I noticed that when using the provided configurations, the results on some datasets differ significantly, particularly for Cornell and Arxiv. Could you kindly update the config files for these datasets?

Additionally, I would greatly appreciate it if you could provide the configuration details used for Table 4 in your paper.

Lastly, I am particularly interested in how you adapted some baseline methods to your setting to ensure consistency, as shown in Table 3, especially GCOPE and All-in-One. Could you share the specific steps, hyperparameters, and configurations used to achieve the reported results?

Below is an example of the issue I encountered when running the provided config on Cornell (10-shot) without any modifications:
`100%|█████████████████████████████████████| 199/199 [00:08<00:00, 24.14it/s, epoch=199, loss=157] 
Dataset cornell Shot 10 Repeat 1: 0.79
In adaptive adj:True
Successfully loaded pre-trained weights!
100%|█████████████████████████████████████| 199/199 [00:07<00:00, 27.29it/s, epoch=199, loss=137]
Dataset cornell Shot 10 Repeat 2: 0.84
In adaptive adj:True
Successfully loaded pre-trained weights!
100%|█████████████████████████████████████| 199/199 [00:07<00:00, 25.95it/s, epoch=199, loss=148]
Dataset cornell Shot 10 Repeat 3: 0.84
In adaptive adj:True
Successfully loaded pre-trained weights!
100%|█████████████████████████████████████| 199/199 [00:08<00:00, 24.38it/s, epoch=199, loss=153]
Dataset cornell Shot 10 Repeat 4: 0.71
In adaptive adj:True
Successfully loaded pre-trained weights!
100%|█████████████████████████████████████| 199/199 [00:07<00:00, 27.48it/s, epoch=199, loss=146]
Dataset cornell Shot 10 Repeat 5: 0.76
Dataset cornell Shot 10: 78.95 | std 4.92
`

Looking forward to your response. Thanks in advance for your help!

**Comment by Cqkkkkkk** [comment]
Thank you for your appreciation. I just reran the codes and was able to replicate the results using the same configuration. Below are the fresh results on Cornell, which are surprisingly higher than those reported in the paper. This discrepancy might be due to differences in the environment or CUDA settings. Since I no longer have access to the original experimental environment, I’m not entirely sure why this happens either. As for scripts beyond the main results, they are currently unavailable due to the broken server. If I have the time to rewrite them or if the server is restored along with the files, I will share them as well.


100%|█████████████████████████████████████████████████████████████████████████| 200/200 [00:06<00:00, 32.70it/s, epoch=199, train_loss=0.0105]
Best ckpt saved at ./ckpts/cornell/dagprompt-GCN-h256.pth
False
Successfully loaded pre-trained weights!
Total tunable parameters: 18436
In adaptive adj:True
Successfully loaded pre-trained weights!
Total tunable parameters: 18734
100%|██████████████████████████████████████████████████████████████████████████████████| 199/199 [00:12<00:00, 16.17it/s, epoch=199, loss=139]
In adaptive adj:True
Successfully loaded pre-trained weights!
Total tunable parameters: 18734
100%|██████████████████████████████████████████████████████████████████████████████████| 199/199 [00:11<00:00, 16.77it/s, epoch=199, loss=137]
In adaptive adj:True
Successfully loaded pre-trained weights!
Total tunable parameters: 18734
100%|██████████████████████████████████████████████████████████████████████████████████| 199/199 [00:11<00:00, 16.79it/s, epoch=199, loss=143]
In adaptive adj:True
Successfully loaded pre-trained weights!
Total tunable parameters: 18734
100%|██████████████████████████████████████████████████████████████████████████████████| 199/199 [00:11<00:00, 16.59it/s, epoch=199, loss=149]
In adaptive adj:True
Successfully loaded pre-trained weights!
Total tunable parameters: 18734
100%|██████████████████████████████████████████████████████████████████████████████████| 199/199 [00:11<00:00, 17.27it/s, epoch=199, loss=135]
Dataset cornell: 89.30 | std 1.44

**Comment by YanJiangJerry** [comment]
Thank you for your prompt response! I fully trust your results and appreciate your contributions. If you get a chance to access the files, please add the config files for graph classification and the code to reproduce any of the baselines, such as All-in-One, GCoPE etc in the same setting to the repository. 
Thanks in advance!
